pipeline/inferrer/inference_manager/src/main/scala/weco/pipeline/inference_manager/adapters/PaletteInferrerAdapter.scala

```diff
@@ -34,15 +38,26 @@ class PaletteInferrerAdapter(host: String, port: Int) extends InferrerAdapter {
     inferrerResponse: Response
   ): InferredData =
     inferrerResponse match {
-      case PaletteInferrerResponse(palette, average_color_hex, params) =>
+      case PaletteInferrerResponse(palette_embedding, average_color_hex) =>
+        val paletteEmbedding = decodeBase64ToFloatList(palette_embedding)
         inferredData.copy(
-          palette = palette,
-          averageColorHex = Some(average_color_hex),
-          binSizes = params.bin_sizes,
-          binMinima = params.bin_minima
+          paletteEmbedding = paletteEmbedding,
+          averageColorHex = Some(average_color_hex)
         )
     }
 
-  implicit val hashParamsDecoder: Decoder[HashParams] = deriveDecoder
+  private def decodeBase64ToFloatList(base64str: String): List[Float] = {
```

Comment:
Let's put this method somewhere common rather than copying and pasting, as it's non-trivial and used for identical cases